.github/workflows/prerelease.yaml

```diff
@@ -0,0 +1,27 @@
+# This config uses industrial_ci (https://github.com/ros-industrial/industrial_ci.git).
+# For troubleshooting, see readme (https://github.com/ros-industrial/industrial_ci/blob/master/README.rst)
+
+name: pre-release
+
+on:
+  workflow_dispatch:
+  pull_request:
```

Comment:
Could you remove pull_request for now?

Reply:
I left this since it is failing. Would you like to merge this anyway? If so I can remove

Reply:
Yeah, let's merge this anyway

Reply:
Removed it.